NeoMathEngine/src/CPU/CpuMathEngineDnn.cpp

```diff
@@ -478,49 +414,37 @@ void CCpuMathEngine::QrnnFPooling( bool reverse, int sequenceLength, int objectS
 	ASSERT_EXPR( result.GetMathEngine() == this );
 	CCpuExecutionScope scope;
 
-	// Global means outside of OMP
-	const float* globalZ = GetRaw( update );
-	const float* globalF = GetRaw( forget );
-	const float* globalH0 = initialState.IsNull() ? nullptr : GetRaw( initialState );
-	float* globalRes = GetRaw( result );
+	const float* z = GetRaw( update );
+	const float* f = GetRaw( forget );
+	const float* h0 = initialState.IsNull() ? nullptr : GetRaw( initialState );
+	float* res = GetRaw( result );
 
 	const int nextObjectOffset = reverse ? -objectSize : objectSize;
 
 	if( reverse ) {
 		const int firstElemOffset = ( sequenceLength - 1 ) * objectSize;
-		globalZ += firstElemOffset;
-		globalF += firstElemOffset;
-		globalRes += firstElemOffset;
+		z += firstElemOffset;
+		f += firstElemOffset;
+		h0 += firstElemOffset;
```

Comment:
`res += firstElemOffset;` instead of `h0`

Reply:
fixed